### ISSUE TITLE
Create Custom Meta Tag Option

### ISSUE THREAD

**Issue by alexfinnarn** [backdrop-contrib/site_verification#6]
Sometimes the meta tag names change and it can be a real pain to have to update a module's code based on how a 3rd party service changes its accepted ways of verification. 

A custom option can be added that allows adding a meta tag and validates input to make sure the meta tag is valid.

**Comment by alexfinnarn** [comment]
Goes along with https://www.drupal.org/project/site_verify/issues/2876765

in Gitter from @kreynen "Since the person who ported that is MIA and we maintain the Drupal project, we should probably move that project namespace back to match https://www.drupal.org/project/site_verify". Where "that" means this "Site Verification" module.

I'm not sure if this module should be renamed or scrapped in favor of porting the Drupal module.

**Comment by deekayen** [comment]
Wow. All you've done so far is complain in the issue queue. Do something that requires my engagement like open a pull request. Maybe I'll hand the module to you to take over.

As for namespace, I intentionally picked a different one because this isn't a direct port of site_verify.

**Comment by alexfinnarn** [comment]
I didn't mean to offend anyone @deekayen but you did make a commit to remove your name from the maintainer list, so I didn't think the module had a current maintainer, nor did anyone else on the contrib team. 

This module was broken in the project installer for any user trying to use that method of installation. I did make an issue and PR for that https://github.com/backdrop-contrib/site_verification/issues/2#issuecomment-406053197. I only waited a couple of days, but since I thought not being able to install the module just due to a packaging issue was a critical issue, I went ahead and repackaged it to fix the issue. 

I will close my issues and move them to a different project which will be the port of the Drupal "site_verify" module. Sorry for any confusion or offence taken.
